.docker/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+# This file is part of the Kimai time-tracking app.
+#
+# For the full copyright and license information, please view the LICENSE
+# file that was distributed with this source code.
+
+version: '2.1'
+services:
+    db:
+        image: mysql:5.6
+        environment:
+            MYSQL_USER: lamp
+            MYSQL_PASSWORD: lamp
+            MYSQL_DATABASE: lamp
+            MYSQL_RANDOM_ROOT_PASSWORD: "yes"
+        volumes:
+            - mysql:/var/lib/mysql
+        restart: always
+        healthcheck:
+            test: ["CMD", "mysqladmin" ,"ping", "-h", "localhost"]
+            # test: ["CMD", "mysql" ,"-u", "lamp", "-plamp", "-h", "localhost", "lamp", "-e", "'desc user;'"]
+            interval: 20s
+            retries: 10
+
+    php:
+        image: kimai
+        build:
+            context: .
+            args:
+                VERSION: 0.5
+                MAILER_FROM: docker@example.com
+                MAILER_URL: null://127.0.0.1
+        links: [ db ]
+        ports:
+            - "9000:9000"
+            - "8001:8001"
+        depends_on:
+            db:
+                condition: service_healthy
+        environment:
+            VERSION: 0.5
+        stdin_open: true
+        tty: true
+        volumes:
+            - ./php.ini:/usr/local/etc/php/php.ini
```

Comment:
can you remember what you changed? I'd prefer to use the maintainers version if possible.

Reply:
It's copied verbatim form the php-fpm docker image and memory limit raised.  The php FPM image does not come with a php.ini but with a dev and production template  I could enable the production one on the image and sed the memory size up if you'd prefer.